### PR TITLE
Finished excercise 1

### PR DESCRIPTION
**Pull request by AnaCDomingo**

### CODE REVIEW

File: 01-JS-DOM-APIs/EX01/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
```
Comment (mtulian) [inline 01-JS-DOM-APIs/EX01/index.html]: **Minor**: Fix the identation here. I don't remember if I told you about minor comments. They are related to have a clean code, the severity associated is low, you can skip this type of comments.

File: 01-JS-DOM-APIs/EX01/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+   	</section>
+   	<section id="hide_Show">
+   		<h2>Hello World!</h2>
+   	</section>
+   	<section>
+   		<p>Sed eu volutpat risus. Pellentesque laoreet vel nisi nec tempus. Nullam id mauris ut metus ultricies vehicula vitae et tellus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Aenean a est sem. Proin eget elementum mi. Nulla iaculis accumsan massa, vel elementum purus vestibulum nec. Mauris fringilla ipsum sit amet dui elementum, in mattis ante hendrerit. Curabitur massa nisi, feugiat eget neque vitae, egestas lacinia felis. Nullam porta, ex sit amet malesuada finibus, arcu nunc scelerisque ante, id fermentum nulla mauris facilisis arcu. Nulla sit amet justo rhoncus, pretium risus in, tristique mauris. Pellentesque at enim venenatis, tempor eros sit amet, feugiat felis. Donec mauris augue, molestie sed urna vel, elementum dictum dui. Cras luctus justo ante, a dapibus enim placerat sit amet. Fusce accumsan libero eget arcu scelerisque sagittis. Pellentesque nec nisl eros.</p>
+   	</section>
+
+   	<script>
+			document.getElementById("hide_Show").style.visibility = "visible";
```
Comment (mtulian) [inline 01-JS-DOM-APIs/EX01/index.html]: Probably you won't see any change once the page is loaded. I mean you will always see the section#hide_show visible. Try to change the approach to use opacity and wait till the page is loaded to make the style update.

File: 01-JS-DOM-APIs/EX01/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+html {
+	font-family: 'Bubbler One', sans-serif;
+	background-color: black;
```
Comment (mtulian) [inline 01-JS-DOM-APIs/EX01/css/styles.css:3]: Try to use classes instead of html tags

Comment (AnaCDomingo) [reply 01-JS-DOM-APIs/EX01/css/styles.css:3]: Will have in consideration.

**Review by mtulian — APPROVED** [review]

**Pull request merged**